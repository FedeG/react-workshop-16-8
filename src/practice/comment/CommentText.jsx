import React from 'react';
import PropTypes from 'prop-types';

const CommentText = ({ children }) => (
  <div className="Comment-text">
    {children}
  </div>
);

CommentText.propTypes = {
  children: PropTypes.element,
};

CommentText.defaultProps = {
  children: null,
};

export default CommentText;
