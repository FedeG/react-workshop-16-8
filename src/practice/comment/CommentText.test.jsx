import React from 'react';
import renderer from 'react-test-renderer';

import CommentText from './CommentText';

describe('CommentText', () => {
  let component;
  let children;

  beforeEach(() => {
    children = <h1>Test</h1>;
    component = renderer.create(
      <CommentText>
        {children}
      </CommentText>,
    );
  });

  test('initial state', () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
