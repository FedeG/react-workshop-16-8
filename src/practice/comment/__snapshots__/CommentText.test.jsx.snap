// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentText initial state 1`] = `
<div
  className="Comment-text"
>
  <h1>
    Test
  </h1>
</div>
`;
